scripts/spack/packages/serac/package.py

```diff
@@ -270,11 +270,17 @@ def cache_name(self):
         if "SYS_TYPE" in env:
             # Are we on a LLNL system then strip node number
             hostname = hostname.rstrip("1234567890")
+        special_case = ""
+        if "+cuda" in self.spec:
```

Comment:
Note: this will affect lassen host configs too, but it might be useful when downloading the docker host configs, since you can easily differentiate between which one is cuda.